packages/website/sidebars.js

```diff
@@ -51,8 +51,8 @@ module.exports = {
         type: 'category',
         label: 'autocomplete-plugin-recent-searches',
         items: [
-          'createLocalStorageRecentSearchesPlugin',
           'createRecentSearchesPlugin',
+          'createLocalStorageRecentSearchesPlugin',
```

Comment:
I changed the order because `createLocalStorageRecentSearchesPlugin` is a specialization of `createRecentSearchesPlugin` so it was odd to have it first.

I assume you've put it first because most users will likely want to use the local storage version. But since I've added an intro that explains the difference and links directly to the local storage version, I think this is fine.